src/number/hash_normalize.cr

```diff
@@ -41,24 +41,14 @@ module Number::HashNormalize
   # XXX: private methods used instead of constants cause current crystal
   # compiler inlines methods better. Looks like it treats constants as
   # real variables, and check for their initialization on every access.
-  private LibCULongIsUInt64 = LibC::ULong == UInt64
-  {% if LibCULongIsUInt64 %}
-    private HASH_BITS = 61
-    private def hash_bits
-      61
-    end
-    private def int_to_hashnorm(v)
-      v.to_i64
-    end
-  {% else %}
-    private HASH_BITS = 31
-    private def hash_bits
-      31
-    end
-    private def int_to_hashnorm(v)
-      v.to_i32
-    end
-  {% end %}
+
+  private HASH_BITS = 61
+  private def hash_bits
+    61
```

Comment:
I add this method to not refer any constant.
See my comment above in this discussion about weird way Crystal handles constants.